livre/core/data/Histogram.h

```diff
@@ -0,0 +1,99 @@
+/* Copyright (c) 2011-2016, EPFL/Blue Brain Project
+ *                          Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+ *
+ * This file is part of Livre <https://github.com/BlueBrain/Livre>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef _Histogram_h_
+#define _Histogram_h_
+
+#include <livre/core/api.h>
+#include <livre/core/mathTypes.h>
+#include <co/distributable.h> // base class
+
+#include <lexis/render/viewHistogram.h> // base class
+
+
+namespace livre
+{
+
+namespace
+{
+const size_t binCount = 256;
+}
+
+class Histogram : public co::Distributable< ::lexis::render::ViewHistogram >
```

Comment:
Since the lexis type is so low level, I'd favor aggregation over inheritance if possible.

Why do you need co::Distributable for this object? I thought that you were sending the histograms to the master node using application specific config events. For that you don't need co::Distributable, which looks heavy weight,  you can declare two friend operators << and >> take a Data{O|I}Stream and a Histogram and then use the basic methods provided by the Collage streams inside. It may be more code, but conceptually simpler to understand in my opinion.